themes/aframe/layout/partials/secondary/sidebar_header.ejs

```diff
@@ -1,12 +1,11 @@
 <%
-
 var links = [
   {
-    url: 'docs/guide/',
+    url: 'docs/' + config.aframe_version + '/guide/',
```

Comment:
I'm confused that [http://localhost:4000/docs/guide/](http://localhost:4000/docs/guide/) -> [http://localhost:4000/master/docs/guide/](http://localhost:4000/master/docs/guide/), though we're linking to the `0.2.0` docs from the homepage "DOCS" link.

<hr>

a few observations/issues:

1. load [http://localhost:4000/docs/master/guide/](http://localhost:4000/docs/master/guide/)
2. notice the "GUIDE" link in the top sidebar nav points to [http://localhost:4000/docs/0.2.0/guide/](http://localhost:4000/docs/0.2.0/guide/)

1. load [http://localhost:4000/](http://localhost:4000/)
2. notice the "DOCS" link in the top-right nav points to [http://localhost:4000/docs/0.2.0/](http://localhost:4000/docs/0.2.0/)

1. load [http://localhost:4000/docs/](http://localhost:4000/docs/)
2. notice [http://localhost:4000/docs/master/guide/](http://localhost:4000/docs/master/guide/) loads

IMO, invalid version numbers should redirect; [http://localhost:4000/docs/0.0.1/guide/](http://localhost:4000/docs/0.0.1/guide/) should take you to [http://localhost:4000/docs/](http://localhost:4000/docs/)